tests/acceptance/features/bootstrap/Sharing.php

```diff
@@ -179,11 +200,22 @@ public function getSharesEndpointPath(?string $postfix = ''):string {
 	public function shareNgGetLastCreatedLinkShareID(): string {
 		$lastResponse = $this->shareNgGetLastCreatedLinkShare();
 		if (!isset($this->getJsonDecodedResponse($lastResponse)['id'])) {
-			throw new Error('Response did not contain share id ' . $this->getJsonDecodedResponse($lastResponse)['id'] . ' for the created public link');
```

Comment:
removed this because if `$this->getJsonDecodedResponse($lastResponse)['id']` is not set it'll probably come undefined. so the error message might not make use of that